ray-on-gke/README.md

```diff
@@ -39,7 +39,7 @@ To submit a Ray job, first establish a connection to the Ray head. For this exam
 to connect to the Ray head via localhost.
 
 ```bash
-$ kubectl port-forward --address 0.0.0.0 service/example-cluster-kuberay-head-svc 8265:8265 &
+$ kubectl port-forward service/example-cluster-kuberay-head-svc 8265 -n ml &
```

Comment:
nit: can you put the `-n` in front of `kubectl`

```
kubectl -n ml port-forward service/example-cluster-kuberay-head-svc 8265 -n ml & 
```

Reply:
Done.